functions/functions/health-check/end-health-check.js

```diff
@@ -0,0 +1,38 @@
+const queryActiveHealthCheck = require('../../database/health-checks/query-active-health-check.dynamodb')
+const queryHealthCheckStatuses = require('../../database/health-statuses/query-health-check-statuses.dynamodb')
+const endHealthCheck = require('../../database/health-checks/end-health-check.dynamodb')
+
+const MAX_POINTS = 2
+
+module.exports.lambda = async (event) => {
+    const {teamId} = JSON.parse(event.body)
+
+    const healthCheck = await queryActiveHealthCheck(teamId)
```

Comment:
This is a good question. But it's an edge case, because you won't be able to call end-health-check function from the frontend if there is no active health check